helpers/utils.py

```diff
@@ -17,6 +17,15 @@ def str_to_list(string):
     return li
 
 
+def str_to_bool(string):
```

Comment:
your best achievement I think

Reply:
hahaha `bool`and `SQLight` in the same sentence is my worst nightmare second only to `nested` and `filter`....